controller/src/lib.rs

```diff
@@ -102,10 +103,13 @@ pub enum Error {
         memory operation ({0:?})"
     )]
     InvalidInterfaceForModule(ManagementInterface),
+
+    #[error("Invalid power state transition")]
+    InvalidPowerStateTransition,
 }
 
 /// An allowed power mode for the module.
-#[derive(Clone, Debug, PartialEq, clap::ValueEnum)]
+#[derive(Clone, Copy, Debug, PartialEq, clap::ValueEnum)]
```

Comment:
Are the comments below still correct?

e.g.
```rust
/// Note: This requires that we never set the `Power_override` bit (SFF-8636
/// rev 2.10a, section 6.2.6, byte 93 bit 2), as that defeats the purpose of
/// hardware control.
```

Reply:
Nope, they're not! Good catch.